server/server.js

```diff
@@ -112,29 +87,26 @@ app.post(`${apiBaseUrl}/employees`,
     },
   }).single('file'),
   asyncMiddleware(async (req, res) => {
-    if (req.fileError) {
-      return res.json({ status: 'Error', message: req.fileError });
-    }
-
+    if (req.fileError) return res.json({ status: 'Error', message: req.fileError });
     const response = await parseAndSaveParticipants(req.file);
     return res.json(response);
   }));
 
+// Get pending users from Keycloak
+app.get(`${apiBaseUrl}/pending-users`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
```

Comment:
should be just MOH here actually, HA approval was descoped

Reply:
Fixed.